.flake8

```
@@ -0,0 +1,21 @@
+# flake8 linter configurations
+# see https://flake8.pycqa.org/en/latest/user/options.html
+
+[flake8]
+# E203 is not PEP8 compliant https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#slices
+# Is excluded from flake8's own config https://flake8.pycqa.org/en/latest/user/configuration.html
+extend-ignore = E203
+max-line-length = 88
+max-doc-length = 88
+per-file-ignores =
+    # rdtools.x.y imported but unused
+    __init__.py:F401
+    # invalid escape sequence '\s'
+    versioneer.py:W605
+exclude =
+    docs
+    .eggs
+    .git
+    .venv
+    build
+    tutorials_and_tools
```

Try specifying the subfolder 
```suggestion
    tutorials_and_tools\tutorials_and_tools
```